repl/tests/tests.rs

```diff
@@ -60,3 +64,120 @@ fn issue_365_run_io_from_command_line() {
     }
     assert_eq!(String::from_utf8_lossy(&output.stdout), "123\n");
 }
+
+
+static COMMAND: &str = "../target/debug/gluon -i";
+static TIMEOUT: u64 = 10_000;
+static PROMPT: &str = "> ";
+
+macro_rules! test {
+    ($e: expr) => { $e.unwrap_or_else(|err| panic!("{}", err)) }
+}
+
+#[test]
+fn prompt() {
+    let mut repl = test!(spawn(COMMAND, Some(TIMEOUT)));
+    test!(repl.exp_string(PROMPT));
+}
+
+#[test]
+fn exit() {
+    let mut repl = test!(spawn(COMMAND, Some(TIMEOUT)));
+    test!(repl.exp_string(PROMPT));
+
+    test!(repl.send_line(":q"));
+    test!(repl.exp_eof());
+}
+
+
+#[test]
+#[ignore] // TODO fix intermittent test failure
```

Comment:
The ``hello_world`` test appears to fail 1/10th of the time with a timeout error and a message containing the expansion of std.io's import and a prompt at the end of the string. Setting #[ignore] and left a TODO note to come back to. Tried ``test!(repl.exp_string(PROMPT));`` before sending another line, also tried letting the process sleep in case it was sending too fast; however, It imports ``std.array`` test without the need to wait/sleep and hasn't experienced any intermittent test failures.

Reply:
You are getting panics sometimes because one of the already existing tests sets an environment variable which adjusts the search path for modules https://github.com/gluon-lang/gluon/pull/466/files#diff-6bf6c6682fd277dbda0194c680c05591R45 . It should just be setting that variable for the command itself instead of globally in the current executable (I think I hacked that in pretty late in the night...).

If you add that environment variable to all the rexpect tests as well they should work consistently.

(The reason for the environment variable is because tests get run from the `./repl` sub folder instead of running from the root folder of the project)

Reply:
Ayyy once again your suggestion worked, and it made travis pass!